docs/clustering.md

```diff
@@ -72,14 +72,23 @@ Recommendations and constraints:
 
 ### Provisioning and Deprovisioning Instances and Groups
 
-* **Provisioning** - Provisioning Instances after installation is supported by updating the `inventory` file and re-running the setup playbook. It's important that this file contain all passwords and information used when installing the cluster, or other instances may be reconfigured (this can be done intentionally).
+* **Provisioning** - Provisioning instances after installation is supported by updating the `inventory` file and re-running the setup playbook. It's important that this file contains all passwords and related information used when installing the cluster; if this is not the case, other instances may be reconfigured (this can be done intentionally).
 
-* **Deprovisioning** - AWX does not automatically de-provision instances since it cannot distinguish between an instance that was taken offline intentionally or due to failure. Instead, the procedure for de-provisioning an instance is to shut it down (or stop the `automation-controller-service`) and run the AWX de-provision command:
+* **Deprovisioning** - Prior to version version 19.3.0, AWX did not automatically deprovision instances since it could not distinguish between an instance that was taken offline intentionally or due to failure. Instead, the procedure for deprovisioning an instance was to shut it down (or stop the `automation-controller-service`) and run the AWX deprovision command:
 
 ```
 $ awx-manage deprovision_instance --hostname=<hostname>
 ```
 
+Starting with AWX version 19.3.0, deprovisioning a node results in one or more Receptor configurations needing to be updated across one or more nodes, which therefore cannot be done via a manual process; the Automation Mesh Installer needs to deprovision the nodes.
```

Comment:
This makes sense, but the reference to the installer may or may not need to be removed, @shanemcd should have a complete answer to that.

This actually brings up a good point that we will need to update https://github.com/ansible/awx-operator/blob/657b5b67db4ddaf9a137219e316e50674b54461d/roles/installer/templates/config.yaml.j2#L231-L260, because we have for sure broken stuff